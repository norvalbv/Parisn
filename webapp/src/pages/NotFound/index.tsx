import React, { ReactElement } from 'react';
import Button from 'components/Button';

const NotFound = (): ReactElement => {
  return (
    <div className="flex h-screen w-full flex-col items-center justify-center gap-4 text-xl">
      Page Not Found :(
      <Button text="Go Home" navigateTo="/" />
    </div>
  );
};

export default NotFound;
