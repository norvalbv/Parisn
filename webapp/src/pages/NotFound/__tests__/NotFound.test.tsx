import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import NotFound from '..';

describe('<NotFound />', () => {
  it('Should render correctly', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <NotFound />
        </BrowserRouter>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
