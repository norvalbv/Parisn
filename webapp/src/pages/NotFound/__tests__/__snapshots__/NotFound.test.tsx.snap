// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> Should render correctly 1`] = `
<div
  className="flex h-screen w-full flex-col items-center justify-center gap-4 text-xl"
>
  Page Not Found :(
  <button
    className="px-6 py-2.5 text-sm text-primary-light bg-primary-dark border rounded font-normal items-center uppercase transition-colors duration-200 hover:bg-primary-neutral/20"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      {
        "width": undefined,
      }
    }
    type="button"
  >
    <span
      className="inline-block tracking-[0.16rem]"
    >
      Go Home
    </span>
  </button>
</div>
`;
